mirroroperator/registrymirror.py

```diff
@@ -399,40 +459,21 @@ def generate_stateful_set(self):
                     spec=client.V1PodSpec(
                         init_containers=[
                             client.V1Container(
-                                name="validate-state-file",
-                                image="python:3.6-alpine",
-                                env=[
-                                    client.V1EnvVar(
-                                        name="STATE_FILE",
-                                        value="/var/lib/registry/scheduler-state.json"
-                                    ),
-                                    client.V1EnvVar(
-                                        name="LOWER_LIMIT",
-                                        value="1024"
-                                    ),
-                                ],
-                                volume_mounts=[
-                                    client.V1VolumeMount(
-                                        name="image-store",
-                                        mount_path="/var/lib/registry"
-                                    )
-                                ],
-                                command=[
-                                    "sh",
-                                    "-e",
-                                    "-c",
-                                    "touch $STATE_FILE; if [[ $(stat -c \"%s\" $STATE_FILE) -lt $LOWER_LIMIT ]]; then rm -f $STATE_FILE; else cat $STATE_FILE | python -m json.tool > /dev/null 2>&1 || rm -f $STATE_FILE; fi"  # noqa
-                                ]
+                                name="generate-ca-certs",
+                                image="cloudbees/docker-certificates:1.2",
+                                command=["/bin/sh"],
+                                args=["-c", script],
```

Comment:
Pretty sure we can do this without this script as there is similar functionality in the image above, but I'd say get this merged and then go back and simplify

see #30